README.md

```diff
@@ -27,4 +27,53 @@ The [code](https://github.com/Microsoft/vsts-urbancode-deploy) is open sourced u
 1. npm update
 1. gulp
 
-The vsix package will be produced in `_package`, and it can be uploaded to Visual Studio Team Services Marketplace for sharing.
\ No newline at end of file
+The vsix package will be produced in `_package`, and it can be uploaded to Visual Studio Team Services Marketplace for sharing.
+
+## FAQ
+### What if I need to work with Sonatype Nexus 3.x?
+Now, the best way to upload artifacts into Nexus 3.x is to use the Maven publish capabilities.
```

Comment:
Nit: 'Now, ' is unnecessary.  Also, no need for 'the' in 'the Maven publish capabilities'.